bitmind/image_transforms.py

```diff
@@ -71,28 +73,207 @@ def __call__(self, img):
         return img
 
 
+# DeeperForensics Distortion Functions
+
+def get_distortion_parameter(distortion_type, level):
```

Comment:
Question/Suggestion: What is the scale/range of these values and how do we interpret? Comment may be helpful